build.sh

```diff
@@ -58,6 +59,13 @@ while [[ "$#" -gt 0 ]]; do
         --infra)
             INFRA="$2"
             shift ;;
+        --amrex)
+            AMREX_INSTALL_PATH="$2"
+            if [ ! -d "${AMREX_INSTALL_PATH}" ]; then
+              echo "--amrex path ${AMREX_PATH} not valid"
+              exit 1
+            fi
```

Comment:
Do you want to check for `${AMREX_INSTALL_PATH}/lib` and `${AMREX_INSTALL_PATH}/include` here? That should catch "oops, I pointed to the wrong place in AMReX" before you get to the actual build step